tutorials/classical_simulation/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p><code>Doppler effect</code> I don't think that "doppler damping" is a thing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>